src/main/io/osd.c

```diff
@@ -308,6 +322,18 @@ static void osdDrawSingleElement(uint8_t item)
 
 #endif // GPS
 
+    case OSD_COMPASS_BAR:
+    {
```

Comment:
This needs a rebase. Once rebased you will see that the functionality for OSD_HOME_DIR has been refactored into a separate function (`osdGetDirectionSymbolFromHeading()`). My suggestion is, since the calculation for OSD_HOME_DIR and this are almost identical, to change it so that the calculation for both cases is done in one function.